contrib/ci/jenkins-community-slave/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM gluon
+
+USER root
```

Comment:
is root inside the container a necessity?

Reply:
Ah sure. We should try this out.

Reply:
OpenWrt will not build as root without setting an additional environment variable.

Reply:
```
configure: error: you should not run configure as root (set FORCE_UNSAFE_CONFIGURE=1 in environment to bypass this check)
```

But we will not add that :)